src/hooks/settingsHooks/useGetSiteUrl.ts

```diff
@@ -0,0 +1,22 @@
+import { useQuery, UseQueryResult } from "react-query"
+
+import { SITE_URL_KEY } from "constants/queryKeys"
+
+import { apiService } from "services/ApiService"
+
+const getSiteUrl = async (siteName: string) => {
+  const settingsResp = await apiService.get(
+    `${process.env.REACT_APP_BACKEND_URL_V2}/sites/${siteName}/settings`
+  )
+  return settingsResp.data.url
+}
```

Comment:
yeap, this is tracked by #1109

Reply:
Right oops I forgot, I will add a separate endpoint called `/sites/:siteName/siteUrl` which will return the live site URL. This is cos the existing `/sites/:siteName/info` endpoint is a little more expensive due to the calls to the GitHub API to get the commits information.

Reply:
Weird but there's also an existing hook in `src/hooks/settingsHooks/useUrlHook.jsx`. Perhaps we can use that directly?

Reply:
didn't use it because it was in js, and i wanted type safety. i didn't migrate the hook over as i'd have to change the call-sites, which is tedious + time-consuming.

could we file an issue for this and let me tackle it down the line?

Reply:
Tracked in #1129.